apps/web/src/pages/Pools/BrowsePoolsPage/BrowsePoolsPage.tsx

```diff
@@ -42,31 +41,30 @@ const flourish = tv({
 });
 
 const BrowsePoolsPage_Query = graphql(/* GraphQL */ `
-  query BrowsePoolsPage($closingAfter: DateTime) {
-    publishedPools(closingAfter: $closingAfter) {
-      id
-      publishingGroup {
-        value
-        label {
-          en
-          fr
+  query BrowsePoolsPage {
```

Comment:
Practically, it makes no difference here since the intention is to pull down all pools.  However, with pagination you really should have an ordering for it to work properly.  You can find the frontend ordering in ActiveRecruitmentSection.